hitl-setup/setup.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# SPDX-FileCopyrightText: 2024 Google LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+FORCE="$1"
+set -xuo pipefail
+HERE=$(realpath "$(dirname "$0")")
+source "$HERE/fpganet" || exit $?
+
+echo Checking that all INTERFACES exist
+for INTERFACE in $INTERFACES; do
+  ip addr show $INTERFACE > /dev/null || exit $?
+done
+echo All INTERFACES exist
+
+FILES="/etc/systemd/system/fpganet.service /etc/fpganet /etc/dhcp/dhcpd.conf /etc/default/isc-dhcp-server"
+EXISTS=""
+for FILE in FILES; do
+  if [ -e "$FILE" ]; then
+    EXISTS="$EXISTS $FILE"
+  fi
+done
+if [ -n "$EXISTS"  -a  "$FORCE" = "-f" ]; then
+  echo 'Running this script with -f will overwrite:'
+  echo "$EXISTS"
+  echo "But now it will stop, because -f wasn't supplied"
+  exit 1
+fi
+
+set -e
+apt-get install -y isc-dhcp-server iproute2
+
+cp "$HERE/fpganet.service" /etc/systemd/system
+cp "$HERE/fpganet" /etc
+
+systemctl daemon-reload
+
+systemctl restart fpganet.service
+systemctl enable fpganet.service
+
+cp "$HERE/dhcpd.conf" /etc/dhcp/
+echo "INTERFACESv4=\"$BRIDGE_NAME\"" > /etc/default/isc-dhcp-server
+
+systemctl add-wants isc-dhcp-server.service fpganet.service
+systemctl restart isc-dhcp-server.service
+systemctl enable isc-dhcp-server.service
```

Comment:
This contains some redundancies:
* `systemctl enable fpganet.service`:
    Start `fpganet.service` on every boot
* `systemctl add-wants isc-dhcp-server.service fpganet.service`:
    If `isc-dhcp-serve.service` starts, also start `fpganet.service`
    `systemctl enable isc-dhcp-server.service`:
    Start `isc-dhcp-server.service` on every boot

Either point makes the other redundant in regard to the starting of `fpganet.service`. I'll delete the add-wants.